### PR TITLE
Removed redundant files. Organized models in namespaces. Changed jQue…

### PR DESCRIPTION
**Pull request by Crevil**
…ry paging.

### CODE REVIEW

File: Core.DomainModel/Course.cs [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+namespace Core.DomainModel
+{
+    public class Course : IEntity
+    {
+        public int Id { get; set; }
+        public string Name { get; set; }
+        public virtual ICollection<Student> Students { get; set; }
+    }
+}
```
Comment (esditm) [inline Core.DomainModel/Course.cs]: new line

File: Core.DomainModel/Student.cs [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    public class Student : IEntity, ICreatedOn, IModifiedOn
+    {
+        public int Id { get; set; }
+        public string Name { get; set; }
+        public DateTime CreatedOn { get; set; }
+        public DateTime ModifiedOn { get; set; }
+        public float AverageGrade { get; set; }
+        public virtual ICollection<Course> Courses { get; set; }
+    }
+}
```
Comment (esditm) [inline Core.DomainModel/Student.cs]: new line

File: Core.DomainModel/Teacher.cs [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+namespace Core.DomainModel
+{
+    public class Teacher : IEntity
+    {
+        public int Id { get; set; }
+        public string Name { get; set; }
+        public virtual ICollection<Course> Courses { get; set; }
+    }
+}
```
Comment (esditm) [inline Core.DomainModel/Teacher.cs]: new line

File: Infrastructure.DataAccess/ChangeSampleSeedInitializer.cs [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+                    context.Database.Connection.Database));
+            base.InitializeDatabase(context);
+        }
+
+        protected override void Seed(SampleContext context)
+        {
+            SeedHelper.Seed(context);
+        }
+    }
+}
```
Comment (esditm) [inline Infrastructure.DataAccess/ChangeSampleSeedInitializer.cs]: new line

File: Infrastructure.DataAccess/SeedHelper.cs [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+                context.Teachers.AddOrUpdate(SeedingHelper.Teacher(context.Courses.ToList()));
+            }
+            context.SaveChanges();
+
+            context.ClassRooms.AddOrUpdate(SeedingHelper.ClassRoom(context.Courses.FirstOrDefault()));
+            context.ClassRooms.AddOrUpdate(SeedingHelper.ClassRoom(context.Courses.ToList().Last()));
+            context.SaveChanges();
+        }
+    }
+}
```
Comment (esditm) [inline Infrastructure.DataAccess/SeedHelper.cs]: new line

File: Presentation.Web/App_Start/EmailService.cs [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+                // Handle errors here.
+                // For example someone typed in a wrong email address.
+                // Right now this check is redundant, as it throws on the next line.
+                throw;
+            }
+
+            return Task.FromResult(0);
+        }
+    }
+}
```
Comment (esditm) [inline Presentation.Web/App_Start/EmailService.cs]: new line

File: Presentation.Web/App_Start/SmsService.cs [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+{
+    public class SmsService : IIdentityMessageService
+    {
+        public Task SendAsync(IdentityMessage message)
+        {
+            // Plug in your SMS service here to send a text message.
+            return Task.FromResult(0);
+        }
+    }
+}
```
Comment (esditm) [inline Presentation.Web/App_Start/SmsService.cs]: new line

File: Presentation.Web/Models/Account/LoginViewModel.cs [hunk 8]
```diff
@@ -0,0 +1,14 @@
...
+    public class LoginViewModel
+    {
+        public string UserName { get; set; }
+        public string Email { get; set; }
+        public string Password { get; set; }
+
+        [Display(Name = "Remember me?")]
+        public bool RememberMe { get; set; }
+    }
+}
```
Comment (esditm) [inline Presentation.Web/Models/Account/LoginViewModel.cs]: new line

File: Presentation.Web/Models/Account/RegisterViewModel.cs [hunk 9]
```diff
@@ -0,0 +1,10 @@
+﻿namespace Presentation.Web.Models.Account
+{
+    public class RegisterViewModel
+    {
+        public string UserName { get; set; }
+        public string Email { get; set; }
+        public string Password { get; set; }
+        public string ConfirmPassword { get; set; }
+    }
+}
```
Comment (esditm) [inline Presentation.Web/Models/Account/RegisterViewModel.cs]: new line

File: Presentation.Web/Models/Manage/ChangePasswordViewModel.cs [hunk 10]
```diff
@@ -0,0 +1,23 @@
...
+        [DataType(DataType.Password)]
+        [Display(Name = "New password")]
+        public string NewPassword { get; set; }
+
+        [DataType(DataType.Password)]
+        [Display(Name = "Confirm new password")]
+        [Compare("NewPassword", ErrorMessage = "The new password and confirmation password do not match.")]
+        public string ConfirmPassword { get; set; }
+    }
+}
```
Comment (esditm) [inline Presentation.Web/Models/Manage/ChangePasswordViewModel.cs]: new line

File: Presentation.Web/Models/Student/IndexViewModel.cs [hunk 11]
```diff
@@ -0,0 +1,12 @@
...
+namespace Presentation.Web.Models.Student
+{
+    public class IndexViewModel
+    {
+        [Display(Name = "Student Name:")]
+        public string Name { get; set; }
+        public int SelectedId { get; set; }
+        public PagedData<NewStudentViewModel> PagedStudents { get; set; } 
+    }
+}
```
Comment (esditm) [inline Presentation.Web/Models/Student/IndexViewModel.cs]: new line

File: Presentation.Web/Models/Student/NewStudentViewModel.cs [hunk 12]
```diff
@@ -0,0 +1,8 @@
+﻿namespace Presentation.Web.Models.Student
+{
+    public class NewStudentViewModel
+    {
+        public int Id { get; set; }
+        public string Name { get; set; }
+    }
+}
```
Comment (esditm) [inline Presentation.Web/Models/Student/NewStudentViewModel.cs]: new line

File: Core.ApplicationServices/Core.ApplicationServices.csproj [hunk 13]
```diff
@@ -12,7 +12,6 @@
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
     <SolutionDir Condition="$(SolutionDir) == '' Or $(SolutionDir) == '*Undefined*'">..\</SolutionDir>
-    <RestorePackages>true</RestorePackages>
```
Comment (esditm) [inline Core.ApplicationServices/Core.ApplicationServices.csproj]: Was this removed automatically?

File: Core.ApplicationServices/Core.ApplicationServices.csproj [hunk 14]
```diff
@@ -57,13 +56,6 @@
     </ProjectReference>
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-  <Import Project="$(SolutionDir)\.nuget\NuGet.targets" Condition="Exists('$(SolutionDir)\.nuget\NuGet.targets')" />
```
Comment (esditm) [inline Core.ApplicationServices/Core.ApplicationServices.csproj]: Was this removed automatically?

File: Core.ApplicationServices/Core.ApplicationServices.csproj (hunk 13 above)
Comment (Crevil) [inline Core.ApplicationServices/Core.ApplicationServices.csproj]: Yes.

File: Core.ApplicationServices/Core.ApplicationServices.csproj (hunk 14 above)
Comment (Crevil) [inline Core.ApplicationServices/Core.ApplicationServices.csproj]: Yes

**Pull request merged**